backend/app/assets/javascripts/spree/backend/stock_movement.js.coffee

```diff
@@ -15,5 +15,5 @@ jQuery ->
         return { results: data.stock_items, more: more }
     formatResult: (stock_item) ->
       variantTemplate({ variant: stock_item.variant })
-    formatSelection: (stock_item) ->
-      "#{stock_item.variant.name} (#{stock_item.variant.options_text})"
+    formatSelection: (stock_item, container, excapeMarkup) ->
```

Comment:
@Mafi88 Not sure why we need to pass these new args ?
Also, there is a typo in `excapeMarkup`, it should be `escapeMarkup`.